Source/WebKit/UIProcess/WebAuthentication/Cocoa/AuthenticationServicesForwardDeclarations.h

```diff
@@ -359,6 +359,59 @@ typedef NSString *ASAuthorizationPublicKeyCredentialResidentKeyPreference;
 
 @end
 
+
+typedef NS_ENUM(NSInteger, ASAuthorizationPlatformPublicKeyCredentialRegistrationRequestStyle) {
+    ASAuthorizationPlatformPublicKeyCredentialRegistrationRequestStyleStandard,
+    ASAuthorizationPlatformPublicKeyCredentialRegistrationRequestStyleConditional,
+};
+
+@interface ASAuthorizationPublicKeyCredentialPRFAssertionOutput: NSObject
+@property (nonatomic, readonly) NSData *first;
+@property (nonatomic, nullable, readonly) NSData *second;
+@end
+
+@interface ASAuthorizationPublicKeyCredentialPRFAssertionInputValues: NSObject
+- (instancetype)initWithSaltInput1:(NSData *)saltInput1 saltInput2:(nullable NSData *)saltInput2;
+@property (nonatomic, readonly) NSData *saltInput1;
+@property (nonatomic, nullable, readonly) NSData *saltInput2;
+@end
+
+@interface ASAuthorizationPublicKeyCredentialPRFAssertionInput: NSObject
+- (instancetype)initWithInputValues:(nullable ASAuthorizationPublicKeyCredentialPRFAssertionInputValues *)inputValues perCredentialInputValues:(nullable NSDictionary<NSData *, ASAuthorizationPublicKeyCredentialPRFAssertionInputValues *> *)perCredentialInputValues;
+@property (nonatomic, nullable, readonly) ASAuthorizationPublicKeyCredentialPRFAssertionInputValues *inputValues;
+@property (nonatomic, nullable, readonly) NSDictionary<NSData *, ASAuthorizationPublicKeyCredentialPRFAssertionInputValues *> *perCredentialInputValues;
+@end
+
+@interface ASAuthorizationPlatformPublicKeyCredentialAssertionRequest (Staging)
```

Comment:
We only use the term "Staging" if the code is intended to be removed at some point. If not, I would just use:

```
@interface ASAuthorizationPlatformPublicKeyCredentialAssertionRequest ()
```

Same comment for the other instances.